examples/amr_human_sample_paired_rds_dag.json

```diff
@@ -0,0 +1,25 @@
+{
```

Comment:
I think we don't necessarily have to check in so many different dags for the amr purposes. Could you just one representative one to keep the examples folder uncluttered.

Reply:
sure!